perf_test/sparse/KokkosSparse_sptrsv_superlu.cpp

```diff
@@ -100,7 +102,7 @@ void print_factor_superlu(int n, SuperMatrix *L, SuperMatrix *U, int *perm_r, in
   }
   printf( "];\n" );
 
-#if 0
+#if defined(KOKKOSKERNELS_ENABLE_TPL_SUPERLU)
```

Comment:
I'm planning to remove these debugging functions in the next PR, but we may get compilation errors/warnings with %d in fprintf using complex scalar type?

Reply:
I didn't encounter any when testing.

Reply:
These "spot-check" are not testing these codes, are they?

Reply:
@e10harvey Bold of you to assume that tests actually enable complex scalar types ;-)

Reply:
> These "spot-check" are not testing these codes, are they?

Correct. The spot-check is not testing these diffs. 

> @e10harvey Bold of you to assume that tests actually enable complex scalar types ;-)

I ran the cholmod and superlu perf tests manually for both double and complex<double> :-)

We can either close or merge this PR -- it's really just an onboarding task for myself and the only real improvement is the super LU perf test argument processing.